src/css/home.scss

```diff
@@ -578,6 +578,25 @@
     }
   }
 
+  .governance-forum {
+    background-color: $grey-10;
+
+    h3 {
+      color: $grey-60;
+      margin: 14px 0 72px;
+    }
+
+    @media (min-width: #{$bp-medium}) {
+      h3 {
+        margin: 14px 0 86px;
+      }
+    }
+
+    .governance-feature-card {
+      height: 500px;
+    }
+  }
+
```

Comment:
Maybe we can extract it to the scss mixin because this snippet of code is duplicated (`join.scss`). Or, add it as a common class outside of the parent class so we don't have to duplicate it.

Reply:
I think you're right. I extracted it to the mixin.